web/public/js/controllers/dojo-event-form-controller.js

```diff
@@ -73,6 +70,12 @@
     $scope.datepicker = {};
     $scope.datepicker.minDate = now;
 
+    $scope.$watch('eventInfo.date', function (startDate) {
+      var newStartDate = angular.copy(startDate);
```

Comment:
does this actually change the startdate of the event? cause I can see it changes the endTime, not the startTime though